lib/bulk-apply-applications.js

```diff
@@ -243,86 +249,143 @@ function bulkApplyApplications (args, callback) {
+              emailParents(cb);
+            } else {
+              return cb();
+            }
+          });
 
-            function emailParents (cb) {
-              if (_.isEmpty(profile.parents)) return cb();
-              var parentsEmailed = [];
-              async.eachSeries(profile.parents, function (parent, cb) {
-                if (!_.isObject(parent)) {
-                  seneca.act({role: 'cd-users', cmd: 'load', id: parent, user: requestingUser}, function (err, parentUser) {
-                    if (err) return cb(err);
-                    payload.to = parentUser.email;
-                    if (!_.contains(parentsEmailed, payload.to) && payload.to !== profile.email) {
-                      parentsEmailed.push(payload.to);
-                      seneca.act({role: 'cd-dojos', cmd: 'send_email', payload: payload}, cb);
-                    } else {
-                      return cb();
-                    }
-                  });
-                } else {
-                  payload.to = parent.email;
+          //email the parent email address
+          function emailParents (cb) {
+            if (_.isEmpty(profile.parents)) return cb();
+            var parentsEmailed = [];
+            async.eachSeries(profile.parents, function (parent, cb) {
+              if (!_.isObject(parent)) {
+                seneca.act({role: 'cd-users', cmd: 'load', id: parent, user: requestingUser}, function (err, parentUser) {
+                  if (err) return cb(err);
+                  payload.to = parentUser.email;
                   if (!_.contains(parentsEmailed, payload.to) && payload.to !== profile.email) {
                     parentsEmailed.push(payload.to);
+                    //send the email
                     seneca.act({role: 'cd-dojos', cmd: 'send_email', payload: payload}, cb);
                   } else {
                     return cb();
                   }
+                });
+              } else {
+                payload.to = parent.email;
+                if (!_.contains(parentsEmailed, payload.to) && payload.to !== profile.email) {
+                  parentsEmailed.push(payload.to);
+                  //send the email
+                  seneca.act({role: 'cd-dojos', cmd: 'send_email', payload: payload}, cb);
+                } else {
+                  return cb();
                 }
-              }, cb);
+              }
+            }, cb);
+          }
+        });
+      });
+      return done(null, profiles, event, session, dojo, tickets, emailCode, dojoEmailCode);
+    }
+
+    function sendDojoEmail (profiles, event, session, dojo, tickets, emailCode, dojoEmailCode, done) {
+      process.nextTick(function () {
+        //if email notifications are enabled for the event, send an email to the dojo
+        if (eventData.emailNotifications) {
+          var emailSubject;
+          //subject for email to send to the dojo
+          emailSubject = dojoEmailSubjectData[applications[0].status];
```

Comment:
If i don't misinterpret, https://github.com/CoderDojo/cp-zen-platform/pull/1116/files#diff-f9d511c45522040c41af45c02e48acaaR100 defines a application status as 'received' while there is no equivalent subject for email.
Instead of relying on the missing translation of subject to avoid sending the email, specifically restrict to which scenario this email is supposed to be sent